src/pages/Sign/sections/SignSubscription.tsx

```diff
@@ -52,26 +54,37 @@ const SIGN_SUBSCRIPTION = gql`
     }
   }
 `
-
-interface SignStatusData {
-  signStatus: {
-    status: {
-      signState: SIGNSTATE
-      collectStatus: {
-        status: BANKIDSTATUS
-        code:
-          | 'started'
-          | 'userSign'
-          | 'noClient'
-          | 'outstandingTransaction'
-          | 'expiredTransaction'
-          | 'certificateErr'
-          | 'userCancel'
-          | 'cancelled'
-          | 'startFailed'
+export const SIGN_QUERY = gql`
+  query SignStatus {
+    signStatus {
+      collectStatus {
+        status
+        code
       }
+      signState
     }
   }
+`
+
+interface CollectStatus {
```

Comment:
Just a tip for the future, I did this in the app: https://github.com/HedvigInsurance/app/pull/279

It handles all this crap for one so that you don't have to write exhaustive types for stuff that is in the introspection query anyways 😄

Reply:
yeh i think that'd be good to have haha